--- /usr/local/lib/python3.11/site-packages/basicsr/data/degradations.py.orig	2025-05-31 16:59:41.284896251 +0000
+++ /usr/local/lib/python3.11/site-packages/basicsr/data/degradations.py	2025-05-31 17:02:02.956293343 +0000
@@ -5,7 +5,7 @@
 import torch
 from scipy import special
 from scipy.stats import multivariate_normal
-from torchvision.transforms.functional_tensor import rgb_to_grayscale
+from torchvision.transforms.functional import rgb_to_grayscale
 
 # -------------------------------------------------------------------- #
 # --------------------------- blur kernels --------------------------- #
